mypy/build.py

```diff
@@ -998,6 +999,162 @@ def generate_deps_for_cache(manager: BuildManager,
     return rdeps
 
 
+def baseline_format_error(
+    baseline_format: str, stdout: TextIO, options: Options
+) -> NoReturn:
+    from mypy import main
```

Comment:
Shouldn't this be imported at the top of the file

Reply:
avoid circular tho